include/net/multicell_location.h

```diff
@@ -41,28 +41,39 @@ enum multicell_service {
 	MULTICELL_SERVICE_POLTE
 };
 
+/** Cellular positioning input parameters. */
+struct multicell_location_params {
+	/** Cellular positioning service to be used. */
+	enum multicell_service service;
+	/** Neighbor cell data. */
+	const struct lte_lc_cells_info *cell_data;
+	/**
+	 * @brief Timeout (in seconds) of how long the cellular positioning procedure can take.
+	 * Zero means that the timer is disabled.
+	 */
+	uint16_t timeout;
```

Comment:
it is 32 in rest_client, maybe some alignments within a stack

Reply:
Based on discussion, I think we decided to leave this as it is. I.e., int32_t (ms) in http_client and rest_client, and uint16_t (s) in multicell_location and location libraries.

Reply:
I'll close this comment as we've changed timeouts to `int32_t`. See more details in comment https://github.com/nrfconnect/sdk-nrf/pull/7978#discussion_r904673363